src/instructlab/eval/mt_bench_branch_generator.py

```diff
@@ -60,7 +60,7 @@ def generate(judge_model_name, branch, taxonomy_dir, output_dir):
                 if q is None or a is None:
                     continue
 
-                c = ex["question"] if "context" in ex else None
+                c = ex["context"] if "context" in ex else None
```

Comment:
nit: This would be the same as `c = ex.get("context")` I think.

Reply:
+1

Reply:
I think this is worth a quick change to avoid any future confusion as well.

Reply:
Done.